composer.json

```diff
@@ -164,7 +163,36 @@
         }
     },
     "scripts": {
-        "post-drupal-scaffold-cmd": "cp web/sites/example.settings.local.php web/sites/default/settings.local.php",
-        "post-create-project-cmd": "Sous\\Starter::installTheme"
+        "post-create-project-cmd": [
```

Comment:
See `sous/starter.php` - We're doing some find-and-replace on `sous-project` so these commands point to our project-specific custom theme